templates/pages/login.html.twig

```diff
@@ -60,27 +60,27 @@
                      </span>
                   {% endif %}
                </label>
-               <input type="password" class="form-control" name="{{ pwdfield }}" placeholder="" autocomplete="off" tabindex="2" />
+               <input type="password" class="form-control" id="login_password" name="{{ pwdfield }}" placeholder="" autocomplete="off" tabindex="2" />
             </div>
 
             {% if constant('GLPI_DEMO_MODE') %}
                <div class="mb-3">
-                  <label class="form-label">{{ __('Language') }}</label>
+                  <label class="form-label" for="dropdown_language1">{{ __('Language') }}</label>
                   {{ languages_dropdown|raw }}
                </div>
             {% endif %}
 
             {% if config('display_login_source') %}
                <div class="mb-3">
-                  <label class="form-label">{{ __('Login source') }}</label>
+                  <label class="form-label" for="dropdown_auth1">{{ __('Login source') }}</label>
```

Comment:
```suggestion
                  <label class="form-label" for="dropdown_auth{{ rand }}">{{ __('Login source') }}</label>
```

Reply:
I do not understand :/ Ids are set from `{{ auth_dropdown_login|raw }}`, this already take care of passed `rand` param as far as I know

Reply:
Ohhh... OK? I understood. With @cedric-anne proposal, this should be ok :)